fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.item.Item;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+/**
+ * See {@link net.minecraft.registry.tag.ItemTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public final class ConventionalItemTags {
+	private ConventionalItemTags() {
+	}
+
+	/**
+	 * Natural stone-like blocks that can be used as a base ingredient in recipes that take stone.
+	 */
+	public static final TagKey<Item> STONES = register("stones");
+
+	// Tool tags
+	public static final TagKey<Item> TOOLS = register("tools");
+	public static final TagKey<Item> SHEARS_TOOLS = register("tools/shears");
+	/**
+	 * For throwable spear weapons, like Minecraft's tridents.
+	 * Note, other throwable weapons like boomerangs and throwing knives are best put into their own tools tag.
+	 */
+	public static final TagKey<Item> SPEARS_TOOLS = register("tools/spears");
```

Comment:
Some people were saying they feel the name `spears` is confusing. But I think it is fine but wanted to double check with folks here too.

For reference, Fabric choose `spears` for their tag name 
https://github.com/FabricMC/fabric/blob/93d8cb82e85c3d5744716430516daef393cf5815/fabric-convention-tags-v1/src/generated/resources/data/c/tags/items/spears.json
Neoforge currently has `tridents`
https://github.com/neoforged/NeoForge/blob/1.20.x/src/generated/resources/data/forge/tags/items/tools/tridents.json

The talk is on whether it is clear enough that `spears` means throwable impaling stick weapons or not. Or if any spears, including non-throwable ones, should be allowed in the tag. or if the tag should be `trident_like`. 

My 2 cents, `spears` is probably fine as long as the javadoc is clear.

Reply:
A trident is a spear, `spears` is the correct generalization of these items, unless you want just trident variants, which would be a separate tag in `spears` at most.

Reply:
I thought some more and yeah `spears` is totally fine. Example, if someone makes a Sword that is throwable like a Trident, it should still go in the `swords` tag as it is a sword, not a trident despite having the same behavior of a trident. These tags are classifying the kind of tool, not the behavior of these tools.